<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Jiiks</author>
 <class>Product</class>
 <widget class="QWidget" name="Product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>68</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">color: #aeaeae;
font-family: &quot;Roboto&quot;;
background: #2d2d30;
background: rgb(30, 30, 38);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="mainContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="icon">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>:/images/zere</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="infoWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="lblInfo">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;BetterDiscord v2.0.0 for Discord&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QStackedWidget" name="rsStack">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageButtons">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="btnGroup" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background: transparent;
	border: 1px solid #30313a;
	color: #fff;
	padding: 7px 10px;
	min-width: 50px;
}

QPushButton::disabled {
	color: #5e5e60;
}

QPushButton::checked,
QPushButton::hover {
	background: rgb(62, 204, 156);
}

QPushButton::checked::disabled {
	background: gray;
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnInstall">
               <property name="styleSheet">
                <string notr="true">border-right: 0;</string>
               </property>
               <property name="text">
                <string>Install</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSkip">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">border-left: 0;
border-right: 0;</string>
               </property>
               <property name="text">
                <string>Skip</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnUninstall">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">border-left: 0;</string>
               </property>
               <property name="text">
                <string>Remove</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageStatus">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblStatus">
            <property name="text">
             <string>Status</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnInstall</sender>
   <signal>clicked(bool)</signal>
   <receiver>Product</receiver>
   <slot>checkBtn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSkip</sender>
   <signal>clicked(bool)</signal>
   <receiver>Product</receiver>
   <slot>checkBtn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUninstall</sender>
   <signal>clicked(bool)</signal>
   <receiver>Product</receiver>
   <slot>checkBtn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkBtn(bool)</slot>
 </slots>
</ui>
